install.sh

```diff
@@ -5,7 +5,7 @@
 # @author  Steve Grunwell
 # @license MIT
 
-DIR=$(dirname "$0")
+DIR="$(pwd)"
```

Comment:
Unfortunately, this won't fix the issue you're seeing in all situations, as `$(pwd)` will give you your present working directory, not the directory the script lives in.

To test it, you can save this script somewhere on your computer:

```sh
#!/usr/bin/env bash

echo $(pwd)
```

The output will differ depending on where you are when you call the script:

```
# From my desktop
~/Desktop $ ./test.sh
/Users/steve/Desktop

# From my home directory.
~/Desktop $ cd ~
~ $ ./Desktop/test.sh
/Users/steve
```